llvm/test/Instrumentation/MemorySanitizer/SPIRV/instrument_global_address_space.ll

```diff
@@ -128,6 +128,7 @@ attributes #2 = { nounwind }
 !77 = !{!"ext_oneapi_bindless_images_sample_2d_usm", i32 79}
 !78 = !{!"ext_oneapi_atomic16", i32 80}
 !79 = !{!"ext_oneapi_virtual_functions", i32 81}
+!79 = !{!"ext_intel_spill_memory_size", i32 82}
```

Comment:
I doubt that this line has any effect on the test, so you may as well drop it. Moreover, I'm not sure what are the rules of redefining metadata, it may not be a valid change at all

Reply:
Yep, I agree with @AlexeySachkov on behalf of sanitizer reviewer.